src/components/ShareTrajectoryModal/index.tsx

```diff
@@ -116,7 +116,7 @@ const ShareTrajectoryModal = ({
             </VerticalFlexbox>
             <VerticalFlexbox gap={10}>
                 <h4> Copy link {Link} </h4>
-                <div className={styles.urlInputContainer}>
+                <HorizontalFlexbox gap={6}>
```

Comment:
Starting to add these in where there are style rules that only specify flex, flex direction, gap etc.